radis/io/hitran.py

```diff
@@ -1409,6 +1466,29 @@ def fetch_hitran(
     # Delete files if needed:
     if cache == "regen":
         ldb.remove_local_files(local_file)
+    else:
+        # Raising AccuracyWarning if local_file exists and doesn't have extra columns in it
+        if ldb.get_existing_files(local_file):
+            import vaex
+
+            df = vaex.open(local_file)
+            extra = ["y_", "gamma_", "n_"]
```

Comment:
Why don't you compare with extra_params only?

And have a special case if extra_params='all'

Reply:
done